src/executables/uber-time.js

```diff
@@ -6,14 +6,15 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 let service = new CommandExecutionService();
 
-program
-  .arguments('<address>')
-  .action(address => {
-    try {
-        service.executeTimeEstimates(address)
-               .then(table => console.log(table));
-    } catch (Error) {
-      console.error('Could not get time estimates');
+program.parse(process.argv);
+
+var address = program.args.toString();
+try {
+    if (!address.length) {
+        throw new TypeError('Address is required');
     }
-  })
-  .parse(process.argv);
+    service.executeTimeEstimates(address)
```

Comment:
Nit: This was actually my fault in terms of formatting - if possible, maybe
```javascript
service.executeTimeEstimates(address).then(table => console.log(table));
```
is ultimately the most readable?